utils/validation.ts

```diff
@@ -0,0 +1,20 @@
+import Joi from 'joi';
```

Comment:
I suggest to create a separate file for user validation, f.e. `userValidation`

Reply:
Fixed